### PR TITLE
[SCB-823] Setup the test in kubernetes

### PR DESCRIPTION
**Pull request by crystaldust**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
- **The JIRA issue is created: [SCB-823](https://issues.apache.org/jira/browse/SCB-823)**
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- **The PR adds the minimal resources to run the demo under a kubernetes environment. With kubernetes and service mesh solutions like [Istio](https://istio.io), it would be much easier to test saga in different size distributed systems and multiple real life senarios like: unstable networking, undetected logic errors etc.**

 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
- **The test is done even though the PR doesn't make any code change**

 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
- **This is not a large contribution.**

---

### CODE REVIEW

**Comment by crystaldust** [comment]
Further works:
- [ ] Pull images from a public registry
- [ ]  Add a tooling service to run tests and collect the results
- [ ]  Simulate different scenarios with Istio, like network delay, undetected server error
- [ ]  More advanced test reports

File: saga-demo/saga-k8s-resources/base/alpha.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: alphaserver
+        version: v1
+    spec:
+      containers:
+      - name: alphaserver
+        image: alpha-server:0.3.0-SNAPSHOT
```
Comment (WillemJiang) [inline saga-demo/saga-k8s-resources/base/alpha.yaml]: Can we use the environment variable to specify the version?

File: saga-demo/saga-k8s-resources/base/alpha.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```
Comment (WillemJiang) [inline saga-demo/saga-k8s-resources/base/alpha.yaml]: Please add ASF License header here.

Comment (crystaldust) [reply saga-demo/saga-k8s-resources/base/alpha.yaml]: I'll add the License header, btw, I'm considering reorganizing the folder structure into a base folder and demos folder. The base folder provide alpha-server and postgresql that all demos need. The demos folder provide the demo's services.

Now I've done the reorganization and provide the spring-demo, pls take a look at the latest commit.

File: saga-demo/saga-k8s-resources/spring-demo/jmeter.configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,265 @@
+apiVersion: v1
```
Comment (WillemJiang) [inline saga-demo/saga-k8s-resources/spring-demo/jmeter.configmap.yaml]: ASF License header.

**Pull request merged**